batchLogReader.py

```diff
@@ -557,17 +556,17 @@ def LogReader(filename,tier,movesets,ratings):
 							sys.stderr.write(str(nicks)+"\n")
 							return False
 						
-				move = line[7+5*spacelog+len(found):string.find(line,"|",7+5*spacelog+len(found))-1*spacelog]
+				move = line[7+5*spacelog+len(found): line.find("|", 7+5*spacelog+len(found)) - 1*spacelog]
```

Comment:
Why did you rename this variable and change the formatting?

Reply:
Format was changed to match how Python is generally written but I can revert it; function was changed because `find` and `rfind` are methods instead of functions in Python 3.

Reply:
OK, I was confused because in Python 2 you'd do `line.find` anyway. But it looks like there is a `find` function on the string module, must be extremely outdated. If they have the same behavior then w/e

The line has a change anyway so probably not worth undoing. But in general for tricky stuff like this, it isn't the time to do other types of work